cdap-ui/app/directives/group-side-panel/group-side-panel.less

```diff
@@ -7,29 +9,68 @@ my-side-panel {
     z-index: 1025;
     width: 40px;
     background-color: white;
-    transition: width 0.5s;
-    transition-duration: 0.5s;
+    .transition(all 0.2s ease);
 
     &.left {
       left: 10px;
+      .btn.btn-info.btn-block {
+        text-align: left;
+      }
+      .item .item-heading {
+        &.item-open {
+          color: white;
+        }
+      }
+      .item-heading {
+        span[class*="icon-"] {
+          font-size: 22px;
+        }
+      }
+      &.expanded {
+        .item-heading {
+          span[class*="icon-"] {
+            font-size: 15px;
+          }
+        }
+      }
     }
 
     &.right {
+      .btn.btn-info.btn-block {
+        text-align: right;
+      }
       right: 10px;
+      .item-heading {
+        span.fa:first-child {
+          font-size: 15px;
```

Comment:
Should we make this consistent? Left side panel has different font size on open and closed and right side panel has different one.